eip/EIPS/eip-4337.md

```diff
@@ -312,6 +313,19 @@ Exceptions to the forbidden opcodes:
 
 ### Reputation scoring and throttling/banning for paymasters
 
+#### Rationale
+
+Paymasters might influence many UserOperations in the mempool.
+To prevent abuse, we "penalize" a paymaster that causes invalidation of large number of UserOperations in the mempool.
+To prevent paymasters from "sybil-attack", we require a paymaster to stake with the system, and thus make such DoS attack very expensive.
```

Comment:
Trying an alternative wording, let me know if it's easier to understand:
A paymaster could invalidate many UserOperations in the mempool with a single state change. This opens a denial-of-service attack vector against bundlers, which would have to re-simulate all of these UserOperations in order to drop them.
To prevent such abuse, the system throttles paymasters that have a high invalidation rate.
This mitigation won't work if the paymaster can be easily redeployed at a new address, aka. "Sybil attack".
To make throttling effective, we require a paymaster to lock a stake, which it can only withdraw after a long delay.  Redeploying to a new address requires either staking an additional amount or waiting a long time.  The stake makes DoS attacks by paymasters expensive and ensures that they don't scale.
This stake is never slashed, and can be withdrawn any time (after unstake delay).

Reply:
added this.